src/common/i18n/locales/en-US.json

```diff
@@ -147,7 +147,11 @@
     "reblog": "Reblog",
     "confirm-title": "Reblog to @{{n}}'s blog?",
     "confirm-ok": "Reblog",
-    "success": "Reblogged!"
+    "success": "Reblogged!",
+    "delete-reblog": "Delete Reblog",
```

Comment:
Let's change strings to these `Undo Reblog`, `Undo reblog?`, `Undo`,  `Reblog removed`